app/assets/stylesheets/ama_layout/layout_components/side_nav.scss

```diff
@@ -1,13 +1,8 @@
 .side-nav{
   background: $white;
   @include grid-column(2,0);
-  @include show-for(large);
```

Comment:
is there a reason we're taking these breakpoints out? I feel like this was maybe there because of tests